tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,121 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+import pytest
+import re
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from threading import Thread
+
+import app.app as app
+
+
+trello_test_organization_id = '5ffd97c7f80c3b13867165e0'
```

Comment:
You shouldn't need an organization id to create a board (it's an optional parameter) - if you do want to use one this should be clear in the readme, and it should probably be an environment variable so that other developers know to update this to their own organization